tests/conftest.py

```diff
@@ -48,7 +48,7 @@ def tmp_dir():
     """
     Fixture: Temporary directory, removed after test completion
     """
-    with tempfile.TemporaryDirectory() as tmp_dir:
+    with tempfile.TemporaryDirectory(dir="./") as tmp_dir:
```

Comment:
What's this for? Is it also why the mounting of volumes didn't work?

Reply:
That is so that `tmp_dir` gets created under `/home/gordo/` and not `/tmp`. I don't recall the mounting volumes problem you mention, but don't think so.

Reply:
ok, but what's it changed for?

Reply:
Ah, the volume bind problem was that I couldn't get the docker python client to bind the volume when using the `gordo` user. 

This here though is just something I noticed that while debugging things.  By doing this, if there is any cruft left over from a failed use `tempfile`, it is local to the repo and more obvious, rather than accumulating in `/tmp`.

Reply:
Getting rid of this.